Assets/Project/Scripts/GamePlayScene/Tile/TileGenerator.cs

```diff
@@ -10,7 +10,16 @@ namespace Project.Scripts.GamePlayScene.Tile
 {
     public class TileGenerator : SingletonObject<TileGenerator>
     {
-        public void CreateTiles(ICollection<TileData> tileDatas)
+        private static Dictionary<ETileType, string> _prefabAddressableKeys = new Dictionary<ETileType, string>()
```

Comment:
```suggestion
        private readonly Dictionary<ETileType, string> _prefabAddressableKeys = new Dictionary<ETileType, string>()
```
`static` じゃなくて良いんじゃないですかね．
https://github.com/LITO-apps/Treevel-client/blob/develop/Assets/Project/Scripts/GamePlayScene/Gimmick/GimmickGenerator.cs#L22

Reply:
確かに